modules/src/cpcontest.py

```diff
@@ -0,0 +1,31 @@
+import requests
+import os
+from templates.text import TextTemplate
+
+
+def process(input, entities):
+    output = {}
+    try:
+
+        r = requests.get('https://cpcontest-api.herokuapp.com/')
```

Comment:
Yes, this is my personal API and I am maintaining it. If I could find any alternative API, which is very well maintained and serves the purpose, I will update it.